foundation/settings.py

```diff
@@ -363,6 +365,8 @@ def _parse_email_list(varname):
         asset_hosts.append('https://%s' % AWS_S3_CUSTOM_DOMAIN)
     else:
         asset_hosts.append('https://s3.amazonaws.com')
+    if CUSTOM_ASSETS_DOMAIN:
+        asset_hosts.append('https://%s' % CUSTOM_ASSETS_DOMAIN)
```

Comment:
@brew I'm merging as it is, but let me know what you think and I'll change things in a different PR if deemed necessary.